app/models/order.rb

```diff
@@ -0,0 +1,3 @@
+class Order < ApplicationRecord
+  belongs_to :menu
```

Comment:
I have doubts about that.
Menu have a lot of meals to be selected. 
How will we know which ones user select through menu?